packages/react/src/Image/ImageStory.module.css

```diff
@@ -0,0 +1,5 @@
+.custom-image {
```

Comment:
Could we rename this file to `Image.stories.module.css` please? We do this in `River`, and could be useful later if we need to pattern match any story specific css files.